packages/editable/src/editable.tsx

```diff
@@ -9,13 +9,8 @@ import {
   useStyles,
   HTMLChakraProps,
 } from "@chakra-ui/system"
-import {
-  createContext,
-  cx,
-  MaybeRenderProp,
-  runIfFn,
-  __DEV__,
-} from "@chakra-ui/utils"
+import { cx, MaybeRenderProp, runIfFn, __DEV__ } from "@chakra-ui/utils"
```

Comment:
As I'm reviewing this, I can see that the definition of the `MaybeRenderProp` exported from `@chakra-ui/utils` is as such:

```ts
export type MaybeRenderProp<P> =
  | React.ReactNode
  | ((props: P) => React.ReactNode)
```

It depends on types from React. Since this is a type, should it also be moved to the `@chakra-ui/react-utils`? I only refactored direct modules, but I am not confident that it was sufficient. Happy to get feedback here as well.

Reply:
Yeah, I think React-specific types should be moved over as well

Reply:
I agree!